Makefile.am

```diff
@@ -3794,6 +3794,7 @@ endif
 if HAVE_GNUTLS
 systemd_journal_remote_LDADD += \
 	$(GNUTLS_LIBS)
+endif
 
 # systemd-journal-remote make sense mostly with full crypto stack
```

Comment:
If the change is appropriate then the comment should be updated to reflect the new reality, not left out of sync.  I'm unsure on the appropriateness of the change though, we should just get gnutls in.